### DISCUSSION TITLE
How to disable mini.cursorword in nvim-tree

### DISCUSSION THREAD

**Discussion by bwafi** [nvim-mini/mini.nvim#498] [Q&A] [answered]
```
local filetype = vim.api.nvim_buf_get_option(0, "filetype")

if filetype == "NvimTree" then
  vim.b.minicursorword_disable = true
end
```

how to turn off mini.cursor word in nvim tree, I don't understand

**Comment by echasnovski** [comment, marked as answer]
What is wrong with approach you supplied? Where do you put it?

Did you try following [documented approach](https://github.com/echasnovski/mini.nvim/blob/1fdbb864e2015eb6f501394d593630f825154385/doc/mini.txt#L299)?

**Reply by bwafi** [reply]
aah sorry I didn't really pay attention,

thank
